app/src/controllers/instance/organizations/sagas.js

```diff
@@ -35,5 +36,5 @@ function* watchFetchOrganizations() {
 }
 
 export function* organizationsSagas() {
-  yield all([watchFetchOrganizations(), organizationSagas(), projectsSagas()]);
+  yield all([watchFetchOrganizations(), organizationSagas(), projectsSagas(), usersSagas()]);
```

Comment:
Why do we watch for `rganizationSagas(), projectsSagas(), usersSagas()` here?
As they are a part of the organization level, I suggest to watch for them inside `controllers/organization`.